packages/node/src/policy-gen/policy-gen-compartment-class.js

```diff
@@ -101,10 +106,15 @@ const updateModuleSource = (moduleDescriptor, canonicalName) => {
  * need to be updated.
```

Comment:
(should) TBH, I no longer understand this description. It needs more context so help me understand the why.